app/controllers/works_controller.rb

```diff
@@ -784,7 +784,11 @@ def load_work
   end
 
   def load_first_chapter
-    @chapter = @work.first_chapter
+    if @work.user_is_owner_or_invited?(current_user) || logged_in_as_admin?
```

Comment:
Did you end up deciding to leave this as is? It's been marked as resolved, but it sounds like you agree with Hound here.

Reply:
Didn't see it, but I've unresolved it and will do it now!